PWGHF/D2H/Tasks/taskDstarToD0Pi.cxx

```diff
@@ -82,6 +87,9 @@ struct HfTaskDstarToD0Pi {
 
   void init(InitContext&)
   {
+    if ((doprocessDataWoML & doprocessDataWML) | (doprocessMcWoMl & doprocessMcWML)) {
```

Comment:
This is a wrong syntax and is the reason for the compilation failure. These are bitwise operators. You have to use bool operators.

Reply:
Plus, you are allowing combinations `doprocessDataWoML` and `doprocessMcWML` and the other way around now, which are also wrong.